addons/hr_holidays/static/src/tours/hr_holidays_tour.js

```diff
@@ -0,0 +1,108 @@
+/** @odoo-module **/
+import tour from 'web_tour.tour';
+import { Markup } from 'web.utils';
+import { _t } from 'web.core';
+import session from 'web.session';
+
+const leaveType = "NotLimitedHR";
+const leaveDateFrom = "01/17/2022";
+const leaveDateTo = "01/17/2022";
+const description = 'Days off';
+
+tour.register('hr_holidays_tour', {
+    url: '/web',
+    rainbowManMessage: _t("Congrats, we can see that your request has been validated."),
+    test: false
+}, [
+    tour.stepUtils.showAppsMenuItem(), 
+    {
+        trigger: '.o_app[data-menu-xmlid="hr_holidays.menu_hr_holidays_root"]',
+        content: Markup(_t("Let's discover the Time Off application")),
```

Comment:
`Markup()` is not needed here (and below) - only use it when you use html tags in the content